mailchimp-connector/backendScripts/actions/batchSubscribe.js

```diff
@@ -0,0 +1,83 @@
+loadScript("/mailchimp-connector/backendScripts/utils.js");
+
+var ExecuteBatchSubscribe = function () {
+    var propertyUtil = Java.type("info.magnolia.jcr.util.PropertyUtil");
+    var standardCharsets = Java.type("java.nio.charset.StandardCharsets");
+    var ioUtils = Java.type("org.apache.commons.io.IOUtils");
```

Comment:
Please move any class init. to the try/catch block (after the logical check of the mime type), because if there is an exception thrown (line 22) then we are wasting resources